lib/src/webvtt/WebVttCue.ts

```diff
@@ -0,0 +1,25 @@
+import type { WebVttRegion } from './WebVttRegion.js';
+
+/**
+ * A WebVTT cue.
+ *
+ * @group WebVTT
+ *
+ * @beta
+ */
+export type WebVttCue = {
+	id: string;
+	startTime: number;
+	endTime: number;
+	pauseOnExit: boolean;
+	text: string;
+	align: AlignSetting;
+	region: WebVttRegion | null;
+	vertical: DirectionSetting;
```

Comment:
This type is defined in TypeScript as `type DirectionSetting = "" | "lr" | "rl";`. Is it expected to use this type, or something else meant to be imported?

Reply:
It is expected to use that type. It is from the `DOM` ts library and is a reference to the [`vertical`](https://developer.mozilla.org/en-US/docs/Web/API/VTTCue/vertical) properties values.

Reply:
Right, the name didn't look aligned, so if it's expected, no concerns anymore